sonar-project.properties

```diff
@@ -19,6 +19,8 @@ sonar.projectKey=apache_incubator-daffodil
 sonar.modules=daffodil-cli,daffodil-core,daffodil-io,daffodil-japi,daffodil-lib,daffodil-macro-lib,daffodil-propgen,daffodil-runtime1,daffodil-runtime1-unparser,daffodil-sapi,daffodil-tdml-lib,daffodil-tdml-processor,daffodil-test,daffodil-test-ibm1,daffodil-udf
 sonar.sources=src/main
 sonar.tests=src/it,src/test
+# Scala/Java APIs contain a lot of duplication by their nature. Best to exclude them.
```

Comment:
There's a JIRA ticket DAFFODIL-1747 which, when fixed, will want to revisit this statement.